docs/proposals/README.md

```diff
@@ -0,0 +1,19 @@
+# OpenShift-Ansible Proposal Process
+
+The following process should be followed when a proposal is needed:
+
+1. Create a pull request with the initial proposal using the proposal template
```

Comment:
Where does the proposal live in tree?  How should it be named?  This is probably obviously, but we might as well spell it out.